coordinator/src/main/scala/com/socrata/datacoordinator/service/Service.scala

```diff
@@ -293,4 +300,43 @@ class Service(serviceConfig: ServiceConfig,
 
 object Service {
   val numThreads = new AtomicInteger
+
+  def apply(serviceConfig: ServiceConfig,
```

Comment:
Is there a reason for adding this `apply()` method?  So much copy and paste.

Reply:
Wanted to be able to create Service as a partially applied function... and then pass the things that need access to zookeeper at the end. but I suppose I can do that all in Main.